src/FSharp.NET.Sdk/FSharp.NET.Sdk.nuspec

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
just remembered, it was because the fsproj doesn't have an output type so `dotnet pack` would fail on it